### DISCUSSION TITLE
Image to Text converter

### DISCUSSION THREAD

**Discussion by Akascape** [TomSchimansky/CustomTkinter#1176] [Showcase]
Changed the whole UI of [Textemage](https://github.com/Akascape/TEXTEMAGE)

![Screenshot 1](https://user-images.githubusercontent.com/89206401/216761210-c70957e6-cff9-4865-bf3a-a68a9426051c.jpg)
![Screenshot 2](https://user-images.githubusercontent.com/89206401/216761211-6aca23b0-aef3-4e57-a420-ddbc6a2b400f.jpg)

**Comment by Tenimes** [comment]
What did you use for extract text from image? Opencv?

**Reply by Akascape** [reply]
Tesseract-OCR
